package.json

```diff
@@ -161,7 +161,7 @@
     "webpack-merge": "^5.8.0"
   },
   "build": {
-    "productName": "EverdomeLauncher",
+    "productName": "OKX Collective Metaverse",
```

Comment:
możliwe że to coś przy podpisywaniu nie zadziała lub updejtach, ale nie jestem pewny na 100%